integration_test/index_test.go

```diff
@@ -66,6 +66,19 @@ func TestKrewIndexAddUnsafe(t *testing.T) {
 	}
 }
 
+func TestKrewIndexAddShowsSecurityWarning(t *testing.T) {
+	skipShort(t)
+
+	test, cleanup := NewTest(t)
+	defer cleanup()
+
+	test.WithEnv(constants.EnableMultiIndexSwitch, 1).WithDefaultIndex()
+	out := string(test.Krew("index", "add", "foo", test.TempDir().Path("index/"+constants.DefaultIndexName)).RunOrFailOutput())
+	if !strings.Contains(out, "WARNING") {
```

Comment:
Can you also check for a more specific message? Just in case when `add` prints two warnings, we want to make sure the test asserts on the correct one.